src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="month month--sun month--31">
+        <div class="month__day"></div>
```

Comment:
```suggestion
        <div class="calendar__day"></div>
```

Reply:
Hi, could I please ask, why the month class name needs to be renamed?